src/librawspeed/decoders/IiqDecoder.cpp

```diff
@@ -187,6 +203,8 @@ RawImage IiqDecoder::decodeRawInternal() {
   mRaw->createData();
 
   DecodePhaseOneC(strips, width, height);
+  if (correction_meta_data.getSize() != 0)
```

Comment:
So if `0x110` is there, but neither of the `0x222`/`0x224` is there, we are simply assuming the 4'th quadrant (the bottom-right one)?

Reply:
It wouldn't necessarily have to be any kind of a quadrant correction.  It could also be bad pixels, flat field, or I think a couple of other things that don't depend on the split row/column